pkgs/games/gdlauncher/default.nix

```diff
@@ -0,0 +1,30 @@
+{ pkgs, lib, ... }:
+let
+  pname = "gdlauncher";
+  version = "1.1.30";
+  name = "${pname}-${version}";
+  src = pkgs.fetchurl {
+    url = "https://github.com/gorilla-devs/GDLauncher/releases/download/v${version}/GDLauncher-linux-setup.AppImage";
+    hash = "sha256-4cXT3exhoMAK6gW3Cpx1L7cm9Xm0FK912gGcRyLYPwM=";
+  };
+
+  appimageContents = pkgs.appimageTools.extractType2 { inherit name src; };
+in
```

Comment:
the let in binding isn't the recommended one anymore.

Reply:
appimageContents in defined in let in so I can use it in appimageTools unless there is a way to define it inside of appimageTools that I don't know about.